components/datetime/src/any/date.rs

```diff
@@ -101,14 +101,14 @@ impl DateFormatter {
     /// ```
     /// use icu::calendar::{any_calendar::AnyCalendar, Date, Gregorian};
     /// use icu::datetime::{options::length, DateFormatter};
-    /// use icu::locid::Locale;
+    /// use icu::locid::{Locale, locale};
     /// use icu_provider::any::DynamicDataProviderAnyMarkerWrap;
     /// use std::str::FromStr;
     ///
     /// let provider = icu_testdata::get_provider();
     ///
     /// let length = length::Date::Medium;
-    /// let locale = Locale::from_str("en-u-ca-gregory").unwrap();
+    /// let locale: Locale = locale!("en-u-ca-gregory");
```

Comment:
Nit, here and elsewhere: you don't need `: Locale` when you are using `locale!`

Reply:
Done.